mindmeld/cli.py

```diff
@@ -37,7 +37,7 @@
 from ._util import blueprint
 from ._version import current as __version__
 from .components import Conversation, QuestionAnswerer
-from .constants import DEVCENTER_URL
+from .constants import DEVCENTER_URL, BINARIES_URL, DUCKLING_VERSION
```

Comment:
No need to keep importing DEVCENTER_URL here, since you just changed the one line in this file that references it.

Reply:
done